lib/store.js

```diff
@@ -83,6 +83,14 @@ Store.prototype.del = function (packet, cb) {
 }
 
 /**
+ * deletes a packet from the store by messageId
+ */
+Store.prototype.delByMessageId = function (messageId) {
+  delete this._inflights[messageId]
+  return this
+}
```

Comment:
If I use `del`, and matches `messageId`, undefined `cb` seems to be called.

https://github.com/redboltz/MQTT.js/blob/c35e0a615b44bb235fecffb6b2ae8bdee4556c5a/lib/store.js#L77

Can I update `del` as follows?

Before
```JavaScript
    cb(null, packet)
```

After
```JavaScript
    if (cb) {
      cb(null, packet)
    }
```